src/args.rs

```diff
@@ -1,8 +1,125 @@
-#[derive(Debug)]
-pub struct Args {}
+use clap::Parser;
+
+#[derive(Parser, Debug, Default)]
+#[command(author, version, about, long_about = None, name = "jperf")]
+pub struct Args {
+	/// Application name
+	#[arg(long)]
+	app: Option<String>,
+
+	/// Cloud name
+	#[arg(long)]
+	cloud: Option<String>,
+
+	/// Unit name
+	#[arg(long)]
+	unit: Option<String>,
+
+	/// Model name
+	#[arg(long)]
+	model: Option<String>,		
+
+	/// Display a large set of metrics
+	#[arg(short, long)]
+	verbose: bool,
```

Comment:
I think verbosity would more naturally refer to log output which I don't know whether we'll need (as this will be a TUI application). I think let's remove this option for now, we can re-add it later if necessary

Reply:
Good idea! Removed. 
For later, do you reckon we would want to display a smaller (or larger) set of plots, one being the default setting? In other words, is TUI configurable for us to select `N` number of displays, or does it accept a single metric at a time? If it lets us display plots for multiple metrics, I thought perhaps we could have a default number, say `2`, and then provide an option to display a larger set of metrics.

Reply:
Sounds all fair! I guess we need to experiment with it and get a feel.